core-bundle/src/ContaoManager/Plugin.php

```diff
@@ -52,13 +50,11 @@ public function getBundles(ParserInterface $parser): array
                 ->setLoadAfter(
                     [
                         FrameworkBundle::class,
-                        SecurityBundle::class,
```

Comment:
I am not sure about this change. The core bundle loads the SchebTwoFactorBundle, which decorates the authentication provider and checks for `security.authenticator.manager` when adding its listeners. How does this still work when the security bundle is loaded after the core bundle?

Reply:
idk, but wouldn't that mean the `SchebTwoFactorBundle` needs `loadAfter` SecurityBundle and not the current statement?